lib/spack/spack/build_environment.py

```diff
@@ -109,7 +109,7 @@ def __call__(self, *args, **kwargs):
         disable = env_flag(SPACK_NO_PARALLEL_MAKE)
         parallel = not disable and kwargs.get('parallel', self.jobs > 1)
 
-        if parallel:
+        if parallel and 'bazel' not in self.command:
```

Comment:
Wait, are you sure Bazel can't build in parallel? Or is the parallel flag just different? @alalazo is right that if the build system **cannot** be parallel, then it doesn't **have** to be defined in this file. But if it can be parallel, there's currently no easier way that I know of to define it's parallel behavior.